src/main/resources/application.yaml

```diff
@@ -61,25 +65,3 @@ spring:
           oidc:
             client-id: ${IA_IDAM_CLIENT_ID:iac}
             client-secret: ${IA_IDAM_SECRET:something}
```

Comment:
again on this, as I mention before maybe for another ticket but we now have these
```suggestion
            client-id: ${WA_IDAM_CLIENT_ID:wa}
            client-secret: ${WA_IDAM_CLIENT_SECRET:something}
```

Reply:
yes, I haven't done any work regarding idam so it is as it is per master.